test/thorest-api/transactions/setup/asserts.js

```diff
@@ -10,6 +10,20 @@ const successfulPostTx = ({ success, body, httpCode }) => {
     expect(body?.id).toBeDefined()
 }
 
+const successfulCallTxNoRevert = ({ success, body, httpCode }) => {
+    expect(success).toBeTrue()
+    expect(httpCode).toBe(200)
+    expect(body?.txID).toBeDefined()
+    expect(body?.vmError).toBe("")
+}
+
+const successfulCallTxRevert = ({ success, body, httpCode }) => {
+    expect(success).toBeTrue()
+    expect(httpCode).toBe(200)
+    expect(body?.txID).toBeDefined()
+    expect(body?.vmError).not.toBe("")
```

Comment:
wonder if we can add specific error we look for ? as this just checks it is not empty